react/src/components/atoms/forms/InputTextFuzzy/index.js

```diff
@@ -73,6 +75,7 @@ class InputTextFuzzy extends React.Component {
           switch (event.key) {
             case 'ArrowDown':
             case 'ArrowUp':
+              event.persist();
```

Comment:
don't think this is necessary unless we want to pass event to a callback, right?